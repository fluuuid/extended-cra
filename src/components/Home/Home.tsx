import React from 'react';
import styled from 'styled-components';
import { Box } from 'theme/ui';

const Logo = styled.div`
  width: 100%;
  max-width: 200px;
`;

const Home: React.FC = () => {
  return (
    <Box>
      <Logo>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 420.623 47.74">
          <path
            fill="#FFF"
            d="M4.896 26.643v18.289c0 1.512-.72 2.232-2.16 2.232h-.648C.72 47.164 0 46.443 0 44.932V3.023A2.735 2.735 0 0 1 2.735.287h26.066c1.44 0 2.232.721 2.232 2.016v.434c0 1.367-.792 2.016-2.232 2.016H4.896v17.57M63.575 44.139V2.232c0-1.514.72-2.232 2.088-2.232h.647c1.44 0 2.16.719 2.16 2.232V42.34h22.467c1.439 0 2.231.719 2.231 2.088v.359c0 1.297-.792 2.088-2.231 2.088H66.311a2.734 2.734 0 0 1-2.736-2.736zM124.414 28.514V2.232c0-1.441.721-2.232 2.088-2.232h.648c1.44 0 2.16.791 2.16 2.232v25.922c0 10.008 4.465 15.193 13.321 15.193 9.001 0 13.321-5.041 13.321-15.266V2.232c0-1.514.576-2.232 2.016-2.232h.864c1.368 0 1.944.719 1.944 2.232v26.066c0 12.6-6.48 19.441-18.289 19.441-11.736.001-18.073-6.696-18.073-19.225zM195.621 28.514V2.232c0-1.441.721-2.232 2.088-2.232h.648c1.44 0 2.16.791 2.16 2.232v25.922c0 10.008 4.465 15.193 13.321 15.193 9.001 0 13.321-5.041 13.321-15.266V2.232c0-1.514.576-2.232 2.016-2.232h.864c1.368 0 1.944.719 1.944 2.232v26.066c0 12.6-6.48 19.441-18.289 19.441-11.736.001-18.073-6.696-18.073-19.225zM266.829 28.514V2.232c0-1.441.721-2.232 2.089-2.232h.647c1.44 0 2.16.791 2.16 2.232v25.922c0 10.008 4.465 15.193 13.321 15.193 9.001 0 13.321-5.041 13.321-15.266V2.232c0-1.514.576-2.232 2.016-2.232h.864c1.368 0 1.944.719 1.944 2.232v26.066c0 12.6-6.48 19.441-18.289 19.441-11.736.001-18.073-6.696-18.073-19.225zM339.477 44.932v-42.7c0-1.441.72-2.232 2.088-2.232h.648c1.439 0 2.16.791 2.16 2.232v42.699c0 1.512-.721 2.232-2.16 2.232h-.648c-1.369.001-2.088-.72-2.088-2.231zM395.133.287h-11.018a2.735 2.735 0 0 0-2.735 2.736v1.364h4.896v.22h8.857c12.817 0 20.449 7.129 20.449 19.01 0 12.025-7.416 18.938-20.521 18.938h-8.785v-.363h-4.896v1.947a2.733 2.733 0 0 0 2.735 2.736h10.657c16.201 0 25.851-8.713 25.851-23.402 0-14.618-9.433-23.186-25.49-23.186z"
          />
        </svg>
      </Logo>
      <p>
        Edit <code>src/components/Home/Home.tsx</code> and save to reload.
      </p>
    </Box>
  );
};

export default Home;
