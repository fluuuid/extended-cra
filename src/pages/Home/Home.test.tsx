import * as React from 'react';
import { shallow } from 'enzyme';
import Home from './Home';

describe('<Home />', () => {
  test('renders', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});
