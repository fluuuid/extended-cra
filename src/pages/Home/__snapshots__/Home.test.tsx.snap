// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> renders 1`] = `ShallowWrapper {}`;
